src/server/test/db/insertMamacReadingsFromCSVTests.js

```diff
@@ -4,56 +4,52 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
+const { mocha, expect, testDB } = require('../common');
 const moment = require('moment');
 const path = require('path');
-
-chai.use(chaiAsPromised);
-const expect = chai.expect;
-
-const recreateDB = require('./common').recreateDB;
-const getDB = require('../../models/database').getDB;
+const Reading = require('../../models/Reading');
 const Meter = require('../../models/Meter');
 const loadMamacReadingsFromCsvFile = require('../../services/loadMamacReadingsFromCsvFile');
 
-const mocha = require('mocha');
-
 mocha.describe('Insert Mamac readings from a file', () => {
-	mocha.beforeEach(recreateDB);
 	let meter;
 	mocha.beforeEach(async () => {
-		await new Meter(undefined, 'Meter', null, false, Meter.type.MAMAC).insert();
-		meter = await Meter.getByName('Meter');
+		conn = testDB.getConnection();
+		await new Meter(undefined, 'Meter', null, false, Meter.type.MAMAC).insert(conn);
```

Comment:
When similar code in `compressedReadingsTests.js` was in a transaction. Do we want this as a transaction or not?